From 757f43c1d47f4b4e67c33d1b1a46a9c1831eb6c2 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Fri, 19 Aug 2022 20:54:15 +0200
Subject: [PATCH 3/6] arm64: dts: rk3399-pinephone-pro: Make LOUT use mono
 differntial mode

Modem configures its codec to use mono differential line input. Configure
the main codec to match.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index 615995f28eb5..719585c4bb5e 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -946,6 +946,7 @@ rt5640: rt5640@1c {
 		clock-names = "mclk";
 		realtek,dmic1-data-pin = <1>;
 		realtek,in3-differential;
+		realtek,lout-differential;
 		#sound-dai-cells = <0>;
 	};
 };
-- 
2.35.3

